openmc/checkvalue.py

```diff
@@ -171,6 +171,34 @@ def check_length(name, value, length_min, length_max=None):
         raise ValueError(msg)
 
 
+def check_increasing(name, value, equality=False):
+    """Ensure that a list's elements are strictly or loosely increasing.
+
+    Parameters
+    ----------
+    name : str
+        Description of value being checked
+    value : collections.Sized
+        Object to check if increasing
+    equality : bool, optional
+        Whether equality is allowed. Defaults to False.
+
+    """
+
+    if equality:
+        for i in range(1, len(value)):
+            if value[i] < value[i-1]:
+                msg = (f'Unable to set "{name}" to "{value}" since its elements must '
+                   f'be loosely increasing"')
```

Comment:
```suggestion
                msg = (f'Unable to set "{name}" to "{value}" since its elements must '
                   'be loosely increasing"')
```

Reply:
I think we don't need the ```f``` formatting char at the start of the 2nd line